crates/sui-protocol-config/src/lib.rs

```diff
@@ -178,11 +181,17 @@ struct FeatureFlags {
     // If true, checks no extra bytes in a compiled module
     #[serde(skip_serializing_if = "is_false")]
     no_extraneous_module_bytes: bool,
+    // Define the value used to round up computation gas charges
+    #[serde(skip_serializing_if = "is_zero")]
+    gas_rounding_step: u64,
```

Comment:
This feels more like a protocol config parameter than feature flag to me.

Reply:
done